nuget/lib/dependabot/nuget/file_updater.rb

```diff
@@ -62,7 +62,9 @@ def try_update_projects(dependency)
 
           next unless project_dependencies.any? { |dep| dep.name.casecmp(dependency.name).zero? }
 
-          NativeHelpers.run_nuget_updater_tool(repo_contents_path, proj_path, dependency, !dependency.top_level?)
+          NativeHelpers.run_nuget_updater_tool(repo_root: repo_contents_path, proj_path: proj_path,
+                                               dependency: dependency, is_transitive: !dependency.top_level?,
+                                               credentials: credentials)
```

Comment:
Just noting that the config patching/restoring is happening in a loop here. I'm guessing the number of project files is tiny in practice so no reason for concern.

Reply:
Correct, there usually aren't a ton of project files.  If it becomes a bottleneck we can move the `NuGet.Config` patching up a level, but for now I wanted to scope it as closely to the CLI tool as possible.